Tests/AppTests/PackageCollectionTests.swift

```diff
@@ -21,7 +21,9 @@ class PackageCollectionTests: AppTestCase {
         do {
             let v = try Version(package: p,
                                 packageName: "Foo",
+                                publishedAt: Date(timeIntervalSince1970: 0),
```

Comment:
FYI: We've also got the `.t0`, `.t1` shortcuts, which are believe are available in all tests.